library/gradle

```diff
@@ -1,6 +1,6 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: fee194c2fda18bcd3d2c633b2d1639c0857515eb
+GitCommit: b277db699944f9fc912baa2e70ba5a13fe4963d6
 
 Tags: 5.6.2-jdk8, 5.6-jdk8, jdk8, 5.6.2-jdk, 5.6-jdk, jdk, 5.6.2, 5.6, latest
```

Comment:
Version numbers in the tag names should be bumped too, right? :sweat_smile:

Reply:
Good catch, yes 🤦‍♂